admin_pages/messages/templates/ee_msg_editor_active_context_element.template.php

```diff
@@ -36,33 +36,31 @@
 $context = esc_attr($context);
 ?>
 
-<div class="context-active-control-container">
-    <div class="activate_context_on_off_toggle_container">
-        <span id="on-off-nonce-<?php echo esc_attr($context); ?>" class="hidden">
-            <?php echo wp_kses($nonce, AllowedTags::getWithFormTags()); ?>
-        </span>
-        <div class="hidden js-data">
-            <span class="ee-active-message"><?php echo wp_kses($active_message, AllowedTags::getAllowedTags()); ?></span>
-            <span class="ee-inactive-message"><?php echo wp_kses($inactive_message, AllowedTags::getAllowedTags()); ?></span>
-        </div>
-        <div class="ee-switch">
-            <?php $checked = $is_active ? 'checked' : ''; ?>
-            <input class="ee-switch__input" <?php echo esc_attr($checked); ?>
-                   data-grpid="<?php echo absint($message_template_group_id); ?>"
-                   id="ee-on-off-toggle-<?php echo esc_attr($context); ?>"
-                   type="checkbox"
-                   value="<?php echo esc_attr($on_off_action); ?>"
-            />
-            <label class='ee-switch__toggle' for="ee-on-off-toggle-<?php echo esc_attr($context); ?>"></label>
-            <div class="ee-switch__label">
-                <span class="ee-switch--ON">
-                    <?php echo wp_kses($active_message, AllowedTags::getAllowedTags()); ?>
-                </span>
-                <span class="ee-switch--OFF">
-                    <?php echo wp_kses($inactive_message, AllowedTags::getAllowedTags()); ?>
-                </span>
-            </div>
+<div class="activate_context_on_off_toggle_container">
+    <span id="on-off-nonce-<?php echo esc_attr($context); ?>" class="hidden">
+        <?php echo wp_kses($nonce, AllowedTags::getWithFormTags()); ?>
+    </span>
+    <div class="hidden js-data">
+        <span class="ee-active-message"><?php echo wp_kses($active_message, AllowedTags::getAllowedTags()); ?></span>
+        <span class="ee-inactive-message"><?php echo wp_kses($inactive_message, AllowedTags::getAllowedTags()); ?></span>
+    </div>
+    <div class="ee-switch">
+        <?php $checked = $is_active ? 'checked' : ''; ?>
+        <input class="ee-switch__input" <?php echo esc_attr($checked); ?>
+               data-grpid="<?php echo absint($message_template_group_id); ?>"
+               id="ee-on-off-toggle-<?php echo esc_attr($context); ?>"
+               type="checkbox"
+               value="<?php echo esc_attr($on_off_action); ?>"
+        />
```

Comment:
I would suggest moving type="checkbox" in front of the input tag in order to have more clear code.